libs/shared/.history/src/consts/providers/provider-credentials_20220815181320.ts

```diff
@@ -0,0 +1,319 @@
+import { CredentialsKeyEnum } from './provider.enum';
+import { IConfigCredentials } from './provider.interface';
+
+const mailConfigBase: IConfigCredentials[] = [
+  {
+    key: CredentialsKeyEnum.From,
+    displayName: 'From email address',
+    description: 'Use the same email address you used to login to your email provider',
+    type: 'string',
+    required: true,
+  },
+  {
+    key: CredentialsKeyEnum.SenderName,
+    displayName: 'Sender name',
+    type: 'string',
+    required: true,
+  },
+];
+
+const smsConfigBase: IConfigCredentials[] = [
+  {
+    key: CredentialsKeyEnum.From,
+    displayName: 'From',
+    type: 'string',
+    required: true,
+  },
+];
```

Comment:
@oba2311 Could you remove those files from the PR? Looks like they are here by mistake :)

Reply:
yup, I probably need to exclude `.history` files... thought  those were exclude by default